pkg/build/inline.go

```diff
@@ -59,24 +57,32 @@ func NewInlinerWithScheme(scheme files.URLScheme, objReader files.FileObjReader)
 		scheme: objReader,
 	}
 	return &Inliner{
-		Readers:      rdrMap,
-		PathRewriter: DefaultPathRewriter,
+		Readers: rdrMap,
 	}
 }
 
-// BundleFiles converts dereferences file-references in for bundle files.
+// BundleFiles converts dereferences file-references in for bundle files. If
```

Comment:
Not really sure what "converts dereferences" means - typo?

Reply:
Done